### ISSUE TITLE
passenger_ruby duplicate path error when starting Nginx with Passenger.

### ISSUE THREAD

**Issue by Nmerey** [domcloud/domcloud#52]
When creating a default Rails application it gives Passenger error:
```
/bin/sh: line 1: /home/defensive-cook-dew/home/defensive-cook-dew/.rvm/rubies/default/bin/ruby: No such file or directory
```
I believe it is due to starting 2 servers one after another one with SSL, somehow it messes up `passenger_ruby` path to duplicate application name e.g `/home/app-name/home/app-name/.rvm/default/wrap/ruby`

_etc/nginx/nginx.conf_
```
...
    expires $expires;
    server {
        server_name _;
        listen 68.183.191.223;
        listen [2400:6180:0:d1::770:4001];
        return 301 https://$host$request_uri;
    }
    server {
        server_name _;
        listen 68.183.191.223:443;
        listen [2400:6180:0:d1::770:4001]:443;
        return 301 https://domcloud.co;
        ssl_certificate /home/domcloud/ssl.combined;
        ssl_certificate_key /home/domcloud/ssl.key;
    }
    include /etc/nginx/conf.d/*.conf;
        server_names_hash_bucket_size 128;
}
```

**Comment by willnode** [comment]
Solved with https://github.com/domcloud/bridge/commit/28a2fabe0eb494ea9215f6ba324450cdfe2847d8

**Comment by Nmerey** [comment]
> Solved with [domcloud/bridge@28a2fab](https://github.com/domcloud/bridge/commit/28a2fabe0eb494ea9215f6ba324450cdfe2847d8)

LGTM
